src/components/icon/icon.tsx

```diff
@@ -65,7 +65,15 @@ export class Icon {
      * will remain the way they look in the file.
      * Therefore we inject the svg as inline markup instead.
      */
-    private renderSvg(svgData: string) {
-        this.host.shadowRoot.querySelector('div.container').innerHTML = svgData;
+    private renderSvg(svgData: string, name: string) {
+        const container = this.host.shadowRoot.querySelector('div.container');
+        if (container) {
+            container.innerHTML = svgData;
+        } else {
+            // tslint:disable-next-line:no-console
+            console.error(
+                `limel-icon: no container found when trying to render svg! icon: ${name}`
+            );
```

Comment:
I don't know why the bug appears, and I only saw it while testing some other stuff. I couldn't see any icons that where missing, and once I had added this to try to find out what the problem was, it never logged anything…

I thought it might be a reasonable thing to have anyway. We could of course throw an error instead of just logging, but make sure to include the icon name in the error message.